src/components/Carousel/Carousel.js

```diff
@@ -111,18 +132,41 @@ class Carousel extends Component {
     });
   }
 
+  renderChildren() {
+    const { children, items } = this.props;
+    const { widthOfSlide } = this.state;
+    const style = { maxWidth: `${widthOfSlide}px` };
```

Comment:
it will also work without px;    
const style = { maxWidth: widthOfSlide };

Reply:
actually, it doesn't :)

I've just checked it

Reply:
done)